client/me/purchases/purchases-list/header/index.jsx

```diff
@@ -43,20 +51,29 @@ const PurchasesHeader = ( { section, translate } ) => {
 					{ translate( 'Payment Methods' ) }
 				</NavItem>
 
-				{ config.isEnabled( 'async-payments' ) && (
+				{ isEnabled( 'async-payments' ) && (
 					<NavItem path={ pendingPayments } selected={ section === 'pending' }>
 						{ translate( 'Pending Payments' ) }
 					</NavItem>
 				) }
 			</NavTabs>
+
+			{ section === 'billing' && (
+				<Search
+					pinned
+					fitsContainer
+					onSearch={ ( term ) => {
+						dispatch( setQuery( 'past', term ) );
+					} }
+					initialValue={ null }
```

Comment:
Is this necessary?

Reply:
Good question. I'll take it out and see if it blows up.